parachain-template/README.md

```diff
@@ -0,0 +1,14 @@
+# Substrate Cumulus Parachain Template
+
+A new [Cumulus](https://github.com/paritytech/cumulus/)-based Substrate node, ready for hacking :cloud:
+
+This project is a fork of the
+[Substrate Node Template](https://github.com/substrate-developer-hub/substrate-node-template)
+modified to include dependencies required for registering this node as a **parathread** or
+**parachain** to an established **relay chain**.
+
+👉 Learn more about parachains [here](https://wiki.polkadot.network/docs/learn-parachains), and
+parathreads [here](https://wiki.polkadot.network/docs/learn-parathreads).
+
+To learn about how to actually use the template to hack together your own parachain check out the
+`README` from the [`substrate-parachain-template` repository](https://github.com/substrate-developer-hub/substrate-parachain-template/).
```

Comment:
What do you think about adding a note here explaining that the template is geared towards Rococo (and thus `ROC`) and an explanatory link for Rococo?